circle.yml

```diff
@@ -42,7 +42,7 @@ jobs:
     steps:
       - checkout
       - <<: *restore_cache
-      - run: ./gradlew classes testClasses
+      - run: ./gradlew classes testClasses -PsmartThingsUserName=$ARTIFACTORY_USERNAME -PsmartThingsPassword=$ARTIFACTORY_PASSWORD
       - <<: *save_cache
       - <<: *persist_to_workspace
 
```

Comment:
Can't comment lower, but the reason the test is failing is because the container isn't exposing the port for goaws to the host correctly. 

I just had to solve this problem here:
https://github.com/SmartThingsOSS/ratpack-sqs/pull/14